tests/functional/openstackoperator_controller_test.go

```diff
@@ -261,6 +280,12 @@ var _ = Describe("OpenStackOperator controller", func() {
 				instance := &ovnv1.OVNController{}
 				g.Expect(th.K8sClient.Get(th.Ctx, names.OVNControllerName, instance)).Should(Not(Succeed()))
 			}, timeout, interval).Should(Succeed())
+
+			// expect the ready status to become unknown since ovn is disabled
```

Comment:
The comment is wrong. We expect the OVN condition to be removed and the Ready condition to be True